packages/reports/tests/unit/routes/reports/new-test.js

```diff
@@ -15,6 +15,7 @@ const NEW_MODEL = {
   createdOn: null,
   request: {
     bardVersion: 'v1',
+    dataSource: 'dummy',
```

Comment:
Maybe add an explicit test somewhere that if `dataSource` wasn't defined it will fall back to default correctly

Reply:
That's actually what this test covers. that line is the expected model. the _newModel test creates a model without passing in a datasource.